build/rpm/dependencies.yaml

```diff
@@ -61,6 +61,12 @@
 'augeasproviders_postgresql':
   :release: '2016'
 
+'docker':
+  :requires:
+    # exclude pupmod-stahnma-epel
```

Comment:
Will this cause a problem?  Is it possible for the user to need epel-related functionality from the docker module in a SIMP system?

Reply:
https://github.com/puppetlabs/puppetlabs-docker/issues/9

I think this feature will be removed and I don't personally agree with epel being managed by this module myself.

It is possible for a user to have missing functionality without this module